### PR TITLE
DOCS-15440 dollar prefixed fields

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15440-dollar-prefixed-fields-v0.9.0/reference/limitations/#general-limitations)

[JIRA](https://jira.mongodb.org/browse/DOCS-15440)

### CODE REVIEW

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM. Cheers.

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -48,6 +48,8 @@ General Limitations
 - The MongoDB Shared Tier is not supported.
 - The :data:`system.js <<database>.system.js>` collection is not
   replicated.
+- Collections with dollar (``$``) prefixed field names are not
```
Comment (XueruiFa) [inline source/reference/limitations.txt:51]: I'm fine with saying that $-prefixed fields will not be supported overall, though technically they should succeed during the CEA phase of replication. We won't be able to test the CEA-only behavior rigorously though.

CC @alan-zheng-mdb for confirmation

**Review by alan-zheng-mdb — CHANGES_REQUESTED** [review]

**Review by davemungo — COMMENTED** [review]
Thanks for the feedback @alan-zheng-mdb

This is ready for another pass.

File: source/reference/limitations.txt (hunk 1 above)
Comment (alan-zheng-mdb) [reply source/reference/limitations.txt]: Can we add a link to the ("$") DOCs?

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM. One small request to link to the related dollar prefix Server DOCs

**Review by davemungo — COMMENTED** [review]
Thanks for the feedback @alan-zheng-mdb  I've added a link. Merging next

**Pull request merged**